custom_components/tuya_local/devices/beok_tgr81_thermostat_c.yaml

```diff
@@ -0,0 +1,145 @@
+name: TGR81 WIFI-EP Controller
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "auto"
+          icon: "mdi:cog"
+        - dps_val: "manual"
+          icon: "mdi:hand-back-right"
+      hidden: true
+    - id: 16
+      type: integer
+      name: temperature
+      range:
+        min: 5
+        max: 90
+      mapping:
+        - scale: 1
+          step: 1
+    - id: 19
+      type: integer
+      name: max_temperature
+      mapping:
+        - scale: 1
+          step: 1
+          range:
+            min: 30
+            max: 90
+    - id: 24
+      type: integer
+      name: current_temperature
+      mapping:
+        - scale: 10
+    - id: 26
+      type: integer
+      name: min_temperature
+      optional: true
+      mapping:
+        - scale: 1
+          step: 1
+          range:
+            min: 5
+            max: 20
+    - id: 101
+      type: integer
+      name: unknown_101
+    - id: 102
+      type: string
+      name: unknown_102
+
+secondary_entities:
+  - entity: sensor

```

Comment:
With other devices, I have not duplicated the current temperature as a sensor, as it is available from the climate device and if you really want it as a separate sensor, you can use a template sensor to extract it.

Reply:
Removed.